tlatools/org.lamport.tlatools/src/tla2sany/semantic/OpApplNode.java

```diff
@@ -97,13 +97,6 @@
      // T.L. it is not clear to me if this array ranges over either of the
      // sets or over both of them. There can be a mixed set of bound variables.
 
-  public SymbolNode subExpressionOf = null ;
```

Comment:
This exact declaration (in name and scope) exists in this class' parent class and was giving my reference attempts problems, so i have removed this declaration.

Reply:
I noticed the duplicated member variable several times over the years but was always to afraid to touch it.  Did you run checks beyond the test suite to make sure removal is safe?

Reply:
I ran `mvn check verify`.

Reply:
After re-reviewing the code base (this double definition occurs in `LabelNode` as well - but i will leave that alone,) i'm satisfied that undoing this presumably unintentional hiding has no detrimental impact.